src/exceptionless.node.ts

```diff
@@ -116,12 +137,16 @@ process.on(BEFORE_EXIT, function (code:number) {
   }
 
   var client = ExceptionlessClient.default;
+  var config = client.config;
   var message = getExitCodeReason(code);
+
   if (message !== null) {
-    client.submitLog(BEFORE_EXIT, message, 'Error')
+    client.submitLog(EXIT, message, 'Error')
```

Comment:
The comment above says only sync code can run but this will trigger a pipeline to run (various chained plugins) which will also enqueue the event). Does this work okay?